R/uvhydrograph-data.R

```diff
@@ -94,7 +94,7 @@ parseUVSupplemental <- function(data, plotName, pts) {
     
     primary_lbl <- getUvLabel(data, "primarySeries")
     date_lbl <- paste(lims_UV$xlim[1], "through", lims_UV$xlim[2])
-    comp_UV_lbl <- data[['comparisonSeries']]$name
+    comp_UV_lbl <- data$reportMetadata$comparisonStationId
```

Comment:
I stole this var since it wasn't being used in its old form.. not sure why it was there in the first place